test.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'my_list'
```

Comment:
- [ ] [REQUIRED] Similarly, you do not have a file name `my_list.rb`, but instead you have `list.rb` file. But you try to import a file with the name `my_list.rb` instead of `list.rb`. This also throws a file load error.